lib/internal/fs/read_file_context.js

```diff
@@ -99,7 +97,7 @@ class ReadFileContext {
     } else {
       buffer = this.buffer;
       offset = this.pos;
-      length = MathMin(kReadFileBufferLength, this.size - this.pos);
+      length = this.size - this.pos;
```

Comment:
```suggestion
      length = this.signal ? MathMin(kReadFileBufferLength, this.size - this.pos) : this.size - this.pos;
```

Reply:
@ronag I am not sure I understand the benefit of this? `this.signal` won't exist when starting the operation?
In fact, after reading the previous discussion I agree that the current implementation has some very important advantages that must be preserved.
One of them is that it is fair and not prone to starvation - if you launch 5 `readFile` with the current 4 threads you won't start reading the fifth one until the first four are completed.